compiler/rustc_target/src/asm/mod.rs

```diff
@@ -880,6 +881,10 @@ impl InlineAsmClobberAbi {
                 "C" | "system" | "efiapi" => Ok(InlineAsmClobberAbi::RiscV),
                 _ => Err(&["C", "system", "efiapi"]),
             },
+            InlineAsmArch::LoongArch64 => match name {
+                "C" | "system" | "efiapi" => Ok(InlineAsmClobberAbi::LoongArch),
```

Comment:
I just noticed this, but this should not include `efiapi`, which is not a valid ABI on loongarch.

Reply:
Thanks for catching that mistake. I'll make the necessary changes.